docs/specs/legacy.yml

```diff
@@ -267,8 +267,8 @@ inputs:
 outputs:
   docs/a.json:
   build.log: |
-    ["warning","reserved-metadata","Metadata 'content_type' is reserved by docfx, remove this metadata","docfx.yml",3,16]
-    ["warning","reserved-metadata","Metadata 'ms.contentlang' is reserved by docfx, remove this metadata","docs/a.md",2,1]
+    ["warning","attribute-reserved","Attribute content_type is reserved for use by Docs. Remove it from your file metadata.","docfx.yml",3,16]
+    ["warning","attribute-reserved","Attribute ms.contentlang is reserved for use by Docs. Remove it from your file metadata.","docs/a.md",2,1]
```

Comment:
that's more like a error message/code for Docs, not for docfx? @yufeih  @herohua

Reply:
yes, #4510 is tracking this, right now my thinking is we can first checkin all the docs message, then gradually move then outside to config files once #4510 is done